phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/data/GunLevelDataConfigProcessor.java

```diff
@@ -12,50 +12,59 @@
 
 import java.util.Collection;
 import java.util.Objects;
+import java.util.Set;
 
+/**
+ * A {@link ConfigProcessor} for {@link GunLevelData}.
+ */
 public class GunLevelDataConfigProcessor implements ConfigProcessor<GunLevelData> {
 
     private static final ConfigProcessor<Key> KEY_PROCESSOR = AdventureConfigProcessors.key();
 
+    private static final ConfigProcessor<Set<Key>> KEY_SET_PROCESSOR = KEY_PROCESSOR.setProcessor();
+
     private static final ConfigProcessor<Collection<Key>> KEY_COLLECTION_PROCESSOR
             = KEY_PROCESSOR.collectionProcessor();
```

Comment:
at this point half of the new ethylene features are motivated by stuff in gun code benefiting from them